helm/jupyterhub/files/dandi_authenticator.py

```diff
@@ -0,0 +1,77 @@
+from kubernetes_asyncio import client
+
+
+def modify_pod_hook(spawner, pod):
+    pod.spec.containers[0].security_context = client.V1SecurityContext(
+        privileged=True
+    )
+    return pod
+
+
+c.KubeSpawner.modify_pod_hook = modify_pod_hook
```

Comment:
might be better sooner than later to copy pre-commit config from other dandi components, e.g. dandi-cli and ensure uniform formatting etc . Here -- all the imports sorting